testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -270,7 +269,7 @@ impl BitcoinRegtestController {
         // Configure UTXO filter
         let pkh = Hash160::from_data(&public_key.to_bytes()).to_bytes().to_vec();
         let address = BitcoinAddress::from_bytes(
-            BitcoinNetworkType::Regtest,
+            self.config.burnchain.get_bitcoin_network().1,
```

Comment:
Not a bug, but if it's not too much trouble, could you unpack this separately?  i.e. 

```
let (_, network_type) = self.config.burnchain.get_bitcoin_network();
/* ... */
network_type
```

Instead of using the tuple index?

Reply:
Addressed with https://github.com/blockstack/stacks-blockchain/pull/1819/commits/221661cd7be83c20e94bf0d34c4864a2ddac5825